src/features/Apollo/ApolloProvider.tsx

```diff
@@ -1,19 +1,19 @@
+'use client';
+
 import { takeshapeAnonymousApiKey, takeshapeApiUrl } from '@/config';
-import { useApollo } from '@/utils/apollo/useApollo';
-import { ApolloProvider as Provider } from '@apollo/client';
+import { createNextSSRApolloClient } from '@/lib/apollo/ssr';
+import { ApolloNextAppProvider } from '@apollo/experimental-nextjs-app-support/ssr';
```

Comment:
This is the new apollo package that provides support for the newer Next features. The main changes are better support for caching and streaming functionality offered by Suspense.